src/python-fastui/fastui/json_schema.py

```diff
@@ -83,6 +83,8 @@ class JsonSchemaInt(JsonSchemaBase, total=False):
     maximum: int
     exclusiveMaximum: int
     multipleOf: int
+    maxLength: int
```

Comment:
I don't think these make sense on the int schema!

Reply:
Oh sorry, this should be at `JsonSchemaString`